ada_2_fhir/fhir/2_fhir_fhir_include.xsl

```diff
@@ -543,6 +543,24 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
             </xsl:otherwise>
         </xsl:choose>
     </xsl:template>
+    <xd:doc>
+        <xd:desc>Transforms ada element of type hoeveelheid to FHIR decimal</xd:desc>
+        <xd:param name="in">ada element may have any name but should have datatype aantal (count)</xd:param>
```

Comment:
datatype hoeveelheid in ada vertaalt zich naar PQ of quantity in HL7v3/FHIR
datatype aantal (count) in ada vertaalt zich inderdaad typisch naar een decimal

ik vind het nogal verwarrend dat dit template een naam heeft dat niet bij de inhoud past en ook de desc lijkt niet te kloppen met de beschrijving van de parameter (en de inhoud van het template dat totaal geen rekening houdt met @unit)

Reply:
This xsl:template was not in use anywhere so I removed it entirely